packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -11,10 +11,14 @@ import 'package:flutter/widgets.dart';
 import 'button_theme.dart';
 import 'chip_theme.dart';
 import 'colors.dart';
+import 'flat_button.dart';
 import 'ink_splash.dart';
 import 'ink_well.dart' show InteractiveInkFeatureFactory;
 import 'input_decorator.dart';
+import 'raised_button.dart';
+import 'slider.dart';
 import 'slider_theme.dart';
+import 'switch.dart';
 import 'typography.dart';
```

Comment:
these imports are unfortunate, but i don't know how else to do it..

Reply:
ya :( I could move that new thing into a new file but then it's just sweeping it under the rug.